<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1593</width>
    <height>1007</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1581</width>
      <height>921</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>pause.jpg</normaloff>pause.jpg</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+X, Ctrl+1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>open.jpg</normaloff>open.jpg</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+O</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>zoom out.jpg</normaloff>zoom out.jpg</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Signal Equalizer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>zoom in.jpg</normaloff>zoom in.jpg</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+-, Ctrl+1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Signal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>save.png</normaloff>save.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+S</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>left.png</normaloff>left.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+E, Ctrl+1</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>781</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>779</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="PlotWidget" name="Channel1_2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>761</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>close.png</normaloff>close.png</iconset>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>right.png</normaloff>right.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+=, Ctrl+1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>play.jpg</normaloff>play.jpg</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+Z, Ctrl+1</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>781</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>779</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="QSlider" name="verticalSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_2">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_5">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_6">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_8">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_9">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_7">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_10">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>10</y>
          <width>22</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Signal</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>781</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>779</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="PlotWidget" name="Channel1_3" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>761</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>110</y>
        <width>691</width>
        <height>281</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>689</width>
         <height>279</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial Spectrogram</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>1392</x>
        <y>80</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Pallette 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pallette 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pallette 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pallette 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pallette 5</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>1260</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Color Pallette</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>1470</x>
        <y>420</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show/Hide</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>420</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>420</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Minimum Frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>460</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Maximum Frequency</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>460</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>540</y>
        <width>691</width>
        <height>281</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>689</width>
         <height>279</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>510</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Updated Spectrogram</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>1470</x>
        <y>850</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show/Hide</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1593</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>open.jpg</normaloff>open.jpg</iconset>
     </property>
     <addaction name="actionChannel1"/>
     <addaction name="actionChannel2"/>
     <addaction name="actionChannel3"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="menuOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionProject"/>
    <addaction name="actionAdd_Toolbar"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionAdd_Files"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionSupport"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuProject"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset>
     <normaloff>new.png</normaloff>new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>save.png</normaloff>save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>close.png</normaloff>close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>undo.jpg</normaloff>undo.jpg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>redo.jpg</normaloff>redo.jpg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionProject">
   <property name="text">
    <string>Show/Hide Spectrogram</string>
   </property>
  </action>
  <action name="actionAdd_Toolbar">
   <property name="text">
    <string>Add Toolbar</string>
   </property>
  </action>
  <action name="actionAdd_Files">
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>Support</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionChannel1">
   <property name="text">
    <string>Channel1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionChannel2">
   <property name="text">
    <string>Channel2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionChannel3">
   <property name="text">
    <string>Channel3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
